test/stripe/account_test.rb

```diff
@@ -289,5 +289,63 @@ class AccountTest < Test::Unit::TestCase
       }
       assert_equal(expected, obj.serialize_params)
     end
+
+    context "#external_source=" do
+      should "can have a token source set" do
+        a = Stripe::Account.new("test_account")
+        a.external_account = "tok_123"
+        assert_equal "tok_123", a.external_account
+      end
+
+      should "set a flag if given an object source" do
+        a = Stripe::Account.new("test_account")
+        a.external_account = {
+          :object => 'card'
```

Comment:
`#external_account=` also supports `object` of type `bank_account` - should there be a test for that too, or is that tested implicitly elsewhere?